### ISSUE TITLE
Can't get into Video settings / Window Translator Errors

### ISSUE THREAD

**Issue by TigerbotHesh** [TigerbotHesh/skin.pm3-hd#7]
Various menu items in the Settings are not working, like Videos, Music and Pictures settings.
The debug log shows the following:

```
ERROR: Window Translator: Can't find window videossettings
ERROR: Window Translator: Can't find window musicsettings
ERROR: Window Translator: Can't find window picturessettings
ERROR: Window Translator: Can't find window weathersettings
ERROR: Window Translator: Can't find window filestackingdialog
ERROR: Window Translator: Can't find window peripherals
ERROR: Window Translator: Can't find window tvsettings
```

**Comment by TigerbotHesh** [comment]
A lot of these were removed with Krypton, see: http://forum.kodi.tv/showthread.php?tid=250862&pid=2325425#pid2325425

**Comment by TigerbotHesh** [comment]
Fixed in @45aaaf44e12f2ffceddfcbd4e5bde861d67a9df8, there may be more stuff missing however.
